tests/test_utils.py

```diff
@@ -224,6 +242,24 @@ def test_user_has_access_via_database_with_context(self):
             'a-test-context'
         )
 
+    def test_user_has_access_via_database_with_all_access_context(self):
+        """
+        Access check should return true if RoleAssignment exists for user.
+        This case handles checking if the role assignement has `ALL_ACCESS_CONTEXT` context.
+        """
+        ConcreteUserRoleAssignment.objects.create(
+            user=self.user,
+            role=self.role
+        )
+
+        with patch('tests.models.ConcreteUserRoleAssignment.get_context', return_value=ALL_ACCESS_CONTEXT):
+            assert user_has_access_via_database(
+                self.user,
+                'coupon-manager',
+                ConcreteUserRoleAssignment,
+                ALL_ACCESS_CONTEXT
```

Comment:
same here - the idea is we ask: do you have this role with this context? and the answer is yes if the user's role context matches exactly or is `*`